backtrace-library/build.gradle

```diff
@@ -77,6 +80,9 @@ dependencies {
     androidTestImplementation 'androidx.test:rules:1.5.0'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    androidTestImplementation 'org.mockito:mockito-core:2.24.5'
```

Comment:
2.28.2 is latest https://mvnrepository.com/artifact/org.mockito/mockito-core/2.28.2